### DISCUSSION TITLE
Help!  Why is there no configuration information on the worker container build.log file

### DISCUSSION THREAD

**Discussion by scottmorgan-payroc** [airbytehq/airbyte#36506] [Connector Questions]
The build.log grows out of control taking over all of the worker container disk space.  How does one throttle or configure max size/rollover of the workers build.log file?

**Comment by scottmorgan-payroc** [comment]
The only relief I could get to the endless disk space consumption was to run 

1. docker exec airbyte-worker rm build.log
2. docker exec airbyte-worker touch build.log

After running above the size stays at zero bytes

**Comment by pedrohsroque** [comment]
Same issue here,
we reached 60GB this week...

**Comment by scottmorgan-payroc** [comment]
With abctl local install I haven’t run into the build.log bloat yet.  My solution under docker compose was to create. Job that deleted the build.log file then used a touch command to create an empty build. log file.  After creating an empty build.log file the file never grew again

Get Outlook for iOS<https://aka.ms/o0ukef>

Scott Morgan  Senior Director, Data Engineering
e: ***@***.***
w: www.payroc.com
________________________________

**Comment by panserjio** [comment]
The same issue, 56 Gb
bash-4.2# ls -ltrh
total 56G
drwxr-xr-x 4 root root 4.0K Jan  2  1970 airbyte-app
-rw------- 1 root root  18M Nov 18  2023 opentelemetry-javaagent.jar
-rw------- 1 root root  28M Dec  7  2023 dd-java-agent.jar
drwxr-xr-x 2 root root 4.0K Feb 24  2024 configs
-rw-r--r-- 1 root root  56G Jan 16 16:56 build.log

[a comment by an automated account is not shown]
